server/controllers/courses/course.js

```diff
@@ -0,0 +1,25 @@
+const getCourseInformation = require('../../database/queries/getCourse');
+
+exports.getAllCourses = (req, res) => {
+  const { id } = req.params;
+  getCourseInformation(id)
+    .then((course) => {
+      if (course.rowCount) {
+        res.status(404).end({
```

Comment:
@AbdallahAmmar96 I think you need `send`, not `end`, since `end` takes no arguments (it will not return any data, so your error message won't make it to the client).